src/modules/flow/iio/nodes.c

```diff
@@ -101,15 +211,15 @@ create_device_channels(struct iio_direction_vector_data *mdata, int device_id)
 {
```

Comment:
For the sake of consistence, I'd change this method name to something like `gyro_create_channels`.